projects/packages/forms/src/contact-form/class-feedback.php

```diff
@@ -880,6 +881,36 @@ private static function geolocate_via_api( $ip_address ) {
 		return $country_code;
 	}
 
+	/**
+	 * Get the browser information from the user agent.
+	 *
+	 * Returns a formatted string like "Chrome (Desktop)" or "Safari (Mobile)".
+	 *
+	 * @return string|null Browser information or null if user agent is not available.
+	 */
+	public function get_browser() {
+		if ( empty( $this->user_agent ) ) {
+			return null;
+		}
+
+		// Use Jetpack Device Detection to parse the user agent.
+		$ua_info = new User_Agent_Info( $this->user_agent );
+
+		// Get browser name.
+		$browser_name = $ua_info->get_browser_display_name();
+
+		// Determine platform type (Mobile, Tablet, or Desktop).
+		$platform_type = 'Desktop';
+		if ( $ua_info->is_tablet() ) {
+			$platform_type = 'Tablet';
+		} elseif ( $ua_info->get_platform() ) {
+			// If there's a mobile platform detected (not false), it's mobile.
+			$platform_type = 'Mobile';
```

Comment:
Needs translation functions?

Reply:
I'm not sure I would translate these strings. In Spanish at least, it would look worse translated because we almost always use the English term (e.g., Chrome Desktop). And it's kind of more technical stat data.